virttest/bootstrap.py

```diff
@@ -350,6 +351,15 @@ def _forced_or_detected(forced, detected):
                                       "u%s" % detected.release)
         arch = _forced_or_detected(options.vt_host_distro_arch,
                                    "Host_arch_%s" % detected.arch)
+        vendor = cpu.get_cpu_vendor_name()
+        revision = ""
+        if 'powerpc' in cpu.get_cpu_arch():
+            revision = cpu.get_cpu_version()
+        elif 'x86_64' or 'i386' in cpu.get_cpu_arch():
```

Comment:
This is always `True`. How about if `cpu.get_cpu_arch() in ('x86_64', 'i386')`?

Reply:
okies..